apps/frontend/src/app/3_organisms/FundingHistoryFrame/FundingHistoryFrame.tsx

```diff
@@ -0,0 +1,207 @@
+    (value: number) => {
+      if (fundingHistory.length < pageSize && value > page) {
+        return;
+      }
+      setPage(value);
+    },
+    [page, fundingHistory.length],
+  );
+
+  const isNextButtonDisabled = useMemo(
+    () => !loading && fundingHistory?.length < pageSize,
+    [loading, fundingHistory],
+  );
+
+  const exportData = useCallback(async () => {
+    const { data } = await getFundingHistory({
+      variables: {
+        user: account,
+        skip: 0,
+        pageSize: EXPORT_RECORD_LIMIT,
+        orderBy: orderOptions.orderBy as BitcoinTransfer_OrderBy,
+        orderDirection: orderOptions.orderDirection,
+      },
+    });
+
+    let funding = data?.bitcoinTransfers || [];
+
+    if (!funding || !funding.length) {
+      addNotification({
+        type: NotificationType.warning,
+        title: t(translations.fundingHistory.actions.noDataToExport),
+        content: '',
+        dismissible: true,
+        id: nanoid(),
+      });
+    }
+    //split each row returned from the graph into 2 rows (part 1 and part 2)
+    const fundingData: FundingHistoryType[] = funding.reduce(
+      (acc: FundingHistoryType[], item) => {
+        const row1 = {
+          timestamp: item.createdAtTimestamp,
+          type: t(translations.fundingHistory.transactionType.part1),
+          sent: item.totalAmountBTC,
+          received: null,
+          serviceFee: null,
+          txHash: item.bitcoinTxHash,
+        };
+
+        const row2 = {
+          timestamp: item.createdAtTimestamp,
+          type: t(translations.fundingHistory.transactionType.part2),
+          sent: null,
+          received: item.totalAmountBTC,
+          serviceFee: item.feeBTC,
+          txHash: item.createdAtTx.id,
+        };
+
+        acc.push(row1, row2);
+        return acc;
+      },
+      [],
+    );
+
+    return fundingData;
+  }, [t, account, addNotification, getFundingHistory, orderOptions]);
+
+  useEffect(() => {
+    setPage(0);
+  }, [orderOptions]);
+
+  return (
+    <>
+      <ExportCSV
+        getData={exportData}
+        filename="transactions"
+        className="mb-7 hidden lg:inline-flex"
+        disabled={!funding || funding.length === 0}
+      />
+      <div className="bg-gray-80 py-4 px-4 rounded">
+        <Table
+          setOrderOptions={setOrderOptions}
+          orderOptions={orderOptions}
+          columns={columnsConfig}
+          rows={fundingHistory}
+          rowTitle={generateRowTitle}
+          isLoading={loading}
+          className="bg-gray-80 text-gray-10 lg:px-6 lg:py-4"
+          noData={t(translations.common.tables.noData)}
+          {...applyDataAttr('funding-history-table')}
+        />
+        <Pagination
+          page={page}
+          className="lg:pb-6 mt-3 lg:mt-6 justify-center lg:justify-start"
+          onChange={onPageChange}
+          itemsPerPage={pageSize}
+          isNextButtonDisabled={isNextButtonDisabled}
+          {...applyDataAttr('funding-history-pagination')}
```

Comment:
```suggestion
          dataAttribute='funding-history-pagination'
```

This was a mistake in other tables, should be as suggested (being updated elsewhere in another PR) - for our UI components we have dataAttribute prop already.